project/src/store/reducer.ts

```diff
@@ -0,0 +1,22 @@
+import {createReducer} from '@reduxjs/toolkit';
+import {fillOffers, changeCity} from './action';
+import {offers} from '../mocks/offers';
+import {CITY} from '../mocks/city';
+
+const initialState = {
+  offers: offers,
```

Comment:
вот здесь изначально в офферах должен быть пустой массив
```suggestion
  offers: [],
```